tests/typecheck/managers/querysets/test_prefetch_related.yml

```diff
@@ -169,34 +173,34 @@
         reveal_type(user.annotated_user) # N: Revealed type is "Any"
         reveal_type(user.to_attr_groups) # N: Revealed type is "builtins.list[django.contrib.auth.models.Group]"
 
-    installed_apps:
-        - django.contrib.auth
+
 
 -   case: prefetch_related_to_attr_conflict_with_annotate
+    installed_apps:
+        - django.contrib.auth
     main: |
         from django.db.models import Prefetch, F
         from django.contrib.auth.models import User, Group
 
         # When mixing `.annotate(foo=...)` and `prefetch_related(Prefetch(...,to_attr=foo))`
         # The last annotate in the chain takes precedence (even if it is prior to the prefetch_related)
         annotate_into_prefetch = (
-            User.objects
+            User.objects # E: Attribute "foo" already defined on "User@AnnotatedWith"  [no-redef]
```

Comment:
I'm not super happy with the format `User@AnnotatedWith`, I would prefer: `User@AnnotatedWith[TypedDict({'foo': Any}]` but I did not found a way to trivially get that, str(...) returns the very verbose `django.contrib.auth.models.User@AnnotatedWith[TypedDict({'foo': Any})]`.

Do you think it's worth adding an utility to better format this ? Or did I missed a mypy util ?

Reply:
Added a suggestion